test/README.expected.md

```diff
@@ -17,7 +17,7 @@
 
 Generate Readme files with a React\-like syntax and package\.json\-aware helpers\.
 
-## Installation
+## 🛠 Installation
```

Comment:
You will have to add the emojis to the other sections in here as well, as the created file will use them. You can find more details when looking at the failed checks of this PR.

Reply:
I'm having a bit of trouble understanding this one. I've added it into README.md.tsx and README.expected.md. Is there somewhere else I need to add it? By 'other sections' do you mean the "Homepage"/"Contributing"/"License" in README.expected.md?

Reply:
Yes! You changed e. g. the `<ExamplesFromPkg>` component so that the heading there contains an emoji. The `<ExamplesFromPkg>` is used in `README.md.tsx` to add an examples section to the README, so the README file that is created there will contain the emoji in the heading. But the `README.expected.md` file still expects a heading without an emoji. Therefore the test that ensures expected and actual files are the same fails.
You can see this in the [test logs](https://github.com/dbartholomae/jsx-readme/pull/39/checks#step:10:244) in lines 244 and 245: it expects Examples without emoji, but gets Examples with emoji.

Reply:
OK I see! I've never worked with a repository with tests like this before so this is really educational, thank you. I'll add other emojis to the rest of the `README.expected.md` file :)